core/store/orm/config_internal_test.go

```diff
@@ -0,0 +1,284 @@
+package orm
```

Comment:
Github makes this look like a new file but it actually just renamed with no changed.

The new file is below.